IBPSA/BoundaryConditions/WeatherData/BaseClasses/getTimeSpanWeatherData.mo

```diff
@@ -0,0 +1,87 @@
+within IBPSA.BoundaryConditions.WeatherData.BaseClasses;
+function getTimeSpanWeatherData
+  "Get the time span of the weather data from the file"
+
+  input String filNam "Name of weather data file";
+  input String tabName "Name of weather table";
+  output Modelica.SIunits.Time[2] timeSpan "start and end time of weather data";
+
+protected
+ String lin "Line that is used in parser";
+ Integer iLin "Line number";
+ Integer index =  0 "Index of string";
+ Integer staInd "Start index used when parsing a real number";
+ Integer nextIndex "dummy for return value of scanInteger / scanReal";
+ Integer nrRows "number of rows in table";
+ Boolean EOF "Flag, true if EOF has been reached";
+ Boolean headerOn "still reading the header";
+ Modelica.SIunits.Time avgIncrement "average time increment of weather data";
+ Modelica.SIunits.Time startTime "end time of weather data";
+ Modelica.SIunits.Time endTime "end time of weather data";
+
+algorithm
+  iLin :=0;
+  EOF :=false;
+  // Get line where table dimensions is defined
+  while (not EOF) and (index == 0) loop
+    iLin:=iLin + 1;
+    (lin, EOF) :=Modelica.Utilities.Streams.readLine(fileName=filNam,
+      lineNumber=iLin);
+    index :=Modelica.Utilities.Strings.find(
+      string=lin,
+      searchString=tabName + "(",
+      startIndex=1,
+      caseSensitive=false);
+  end while;
+  assert(not EOF, "Error: Did not find definition of table" + tabName + " when scanning the weather file."
+                      + "\n   Check for correct weather file syntax.");
+  // Get number of rows
+  staInd :=index + Modelica.Utilities.Strings.length(tabName) + 1;
+  (nrRows, nextIndex) :=Modelica.Utilities.Strings.scanInteger(string=lin,
+  startIndex=staInd);
+
+  assert(nrRows > 1, "Error: Just " + String(nrRows) +" row in table " + tabName + " when scanning the weather file."
```

Comment:
It would be useful to write the name of the weather data file so the users quickly sees which file fails in case multiple files are read

Reply:
Ok. Done.